src/Angular.js

```diff
@@ -1454,8 +1454,7 @@ function allowAutoBootstrap(document) {
   var link = document.createElement('a');
   link.href = src;
   var scriptProtocol = link.protocol;
-  var docLoadProtocol = document.location.protocol;
-  if (docLoadProtocol === scriptProtocol) {
+  if (document.location.origin === link.origin) {
```

Comment:
+1, this looks too risky and doesn't seem to add that much value. Let's deal with this once we have a concrete example where the original change is not sufficient.